src/Runner.Common/BrokerServer.cs

```diff
@@ -92,7 +92,7 @@ public Task ForceRefreshConnection(VssCredentials credentials)
 
         public bool ShouldRetryException(Exception ex)
         {
-            if (ex is AccessDeniedException ade && ade.ErrorCode == 1)
+            if (ex is AccessDeniedException ade)
```

Comment:
I don't think we'd want to retry regular 403s?

Reply:
+1 403 is conclusive

Reply:
Unclear who sets `ErrorCode = 1` today. Do we need to worry about back-compat or is this change OK to make?